tests/integration/conftest.py

```diff
@@ -150,3 +152,43 @@ def delete_namespace():
         k8s.CoreV1Api().delete_namespace("bodywork-dev")
 
     request.addfinalizer(delete_namespace)
+
+
+@fixture(scope="function")
+def add_secrets(request: FixtureRequest) -> None:
+    run(
+        [
+            "kubectl",
+            "create",
+            "secret",
+            "generic",
+            f"--namespace={BODYWORK_DEPLOYMENT_JOBS_NAMESPACE}",
+            "test-bodywork-test-project-credentials",
```

Comment:
Would `bodywork-test-project-credentials` not work as a name?

Reply:
I understand now - maybe something like `secretsgroup-bodywork-test-project-credentials`?

Reply:
I think test_secrets would be a better name that's also more descriptive so you'll know that these secrets are to do with tests when looking at the Bodywork namespace and also to avoid secrets_group=secretsgroup in the test project YAML which seems a bit off.

Reply:
Agreed.